src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java

```diff
@@ -280,6 +284,7 @@ public Proc launch(ProcStarter starter) throws IOException {
                 // find container working dir
                 KubernetesSlave slave = (KubernetesSlave) node;
                 FilePath containerWorkingDirFilePath = starter.pwd();
+                workspace = containerWorkingDirFilePath;
```

Comment:
Note that this use of instance state presumes that a given `ContainerExecDecorator` is used just once, or at least that if it is used multiple times, these are serially. In principle

```groovy
container('xxx') {
  parallel a: {
    bat '…'
  }, b: {
    bat '…'
  }
}
```

might misbehave. This is not generally something you would do (if you actually needed parallelism, you would use multiple pods).